lib/client/db/dbcmd/dbcmd.go

```diff
@@ -568,6 +578,10 @@ func (c *CLICommandBuilder) getSQLServerCommand() *exec.Cmd {
 		args = append(args, "-d", c.db.Database)
 	}
 
+	if c.isSqlcmdAvailable() {
+		return exec.Command(sqlcmdBin, args...)
```

Comment:
Does it support exactly the same arguments as `mssql-cli`?

Reply:
Yes, and since we're always using a tunneled proxy, there is no need to have additional flags.

Reply:
Sounds good, adding a comment to that effect might be helpful here.

Reply:
There is a comment on the function godoc.